frontend/pages/ManageControlsPage/Scripts/components/DeleteScriptModal/DeleteScriptModal.tsx

```diff
@@ -18,12 +21,24 @@ const DeleteScriptModal = ({
   onCancel,
   onDelete,
 }: IDeleteScriptModalProps) => {
+  const { renderFlash } = useContext(NotificationContext);
+
+  const onClickDelete = async (id: number) => {
+    try {
+      await scriptAPI.deleteScript(id);
+      renderFlash("success", "Successfully deleted!");
+    } catch {
+      renderFlash("error", "Couldn’t delete. Please try again.");
+    }
+    onDelete();
```

Comment:
Double-checking: Do you mean for this to be called for both a successful delete and when you catch an error?

Reply:
yes. the parent handles dismissing the modal so it needs to know when the deletion is done.

Reply:
it makes more sense after changing to `onDone`